src/stylesheets/containers/visualiser.scss

```diff
@@ -1,8 +1,59 @@
 @import '../variables.scss';
 
 .visualiser {
-    background: $input-background;
-    height: 100%;
-    padding: 1.2rem;
-    border-radius: $border-radius;
-}
\ No newline at end of file
+  background: $input-background;
+  height: 100%;
+  padding: 1.2rem;
+  border-radius: $border-radius;
+}
+
+.row {
+  width: 97%;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  padding: 12px;
+  align-items: center;
+  &:hover {
+    background: #1e2f3e;
+    cursor: pointer;
+  }
+  .json-content {
+    pre {
+      font-family: Fira Code;
+      font-size: 14px;
+      max-width: 600px;
+      overflow: hidden;
+      white-space: nowrap;
+      text-overflow: ellipsis;
+    }
+  }
+  .json-content-expanded {
+    pre {
+      font-family: Fira Code;
+      font-size: 14px;
+      max-width: 600px;
+      overflow: scroll;
+      white-space: pre;
+      overflow-wrap: break-word;
+    }
+  }
+  .date-time {
```

Comment:
We will use dates a lot makes sense to keep somewhere common or in a separate `date-picker.scss` file. What do you think?

Reply:
We have a date-picker component. This is specific to the date option inside search area